lib/dradis/plugins/veracode/field_processor.rb

```diff
@@ -4,14 +4,16 @@ module Veracode
       class FieldProcessor < Dradis::Plugins::Upload::FieldProcessor
         def post_initialize(args = {})
```

Comment:
This whole block is _so_ ugly 🤮

Reply:
Updated to use `case` ✅

Reply:
In order for the switch statement to work we'd have to do:

```
case
when data.is_a?(::Veracode::Flaw) || data.is_a?(::Veracode::Evidence) || data.is_a?(::Veracode::Vulnerability)
  data
when data['dradis_type'] == 'evidence'
 ...
end
```

Otherwise the original if statement works

Reply:
Good catch!